chain/client/src/info.rs

```diff
@@ -538,9 +538,9 @@ impl InfoHelper {
 
     fn log_chain_processing_info(&mut self, client: &crate::Client, epoch_id: &EpochId) {
         let chain = &client.chain;
-        let use_colour = matches!(self.log_summary_style, LogSummaryStyle::Colored);
+        let use_color = matches!(self.log_summary_style, LogSummaryStyle::Colored);
```

Comment:
As a British resident I strongly disagree ;)

Reply:
(I'm fine with it, just don't tell the king)

Reply:
Heh 😅 turns out `ansi_term` was written by british people, and `yansi` by americans, or so it seems :p